Makefile

```diff
@@ -68,7 +68,8 @@ ifeq ($(TARGET), openshift)
 	rm -rf ./cmd/$(TARGET)/operator/kodata/tekton-pruner
 	rm -rf ./cmd/$(TARGET)/operator/kodata/tekton-addon/pipelines-as-code
 	rm -rf ./cmd/$(TARGET)/operator/kodata/tekton-addon/addons/02-clustertasks/source_external/
-	rm -rf ./cmd/$(TARGET)/operator/kodata/tekton-addon/addons/07-ecosystem/task-*
```

Comment:
>  Why don't we remove the directory completely?

We cannot remove `07-ecosystem`  folder because it have role and rolebinding

> @savitaashture we had remove the files with wildcard (task-*), but now with a single name (tasks)
Seems we are not removing the resources with prefix task-* ?

I am assuming this removal( I mean make clean) is for dev work right so do we need to handle :thinking:

Reply:
I am not sure about the change. I hope this change handled well. Thanks!